web-ui/src/main/resources/catalog/components/search/resultsview/partials/viewtemplates/editor.html

```diff
@@ -53,8 +53,14 @@
                     class="text-muted gn-status"
                     data-ng-class="{'text-success': md.mdStatus == 2, 'text-warning': md.mdStatus == 4}"
                     data-ng-if="md.mdStatus < 50 && isMdWorkflowEnable"
-                    >{{::md.statusWorkflow | getStatusLabel}}</small
                   >
+                    <span data-ng-if="md.draft === 'e'"
```

Comment:
Not sure if related to this change, but the editor board labels, except for `Approved (with working copy)` doesn't show up properly.

Also for the approved with working copy, I don't see the `Working copy` badge as previously.

![editorboard-labels](https://github.com/geonetwork/core-geonetwork/assets/1695003/bf3b2b44-0624-44c5-99e2-4b95871ade8f)

Reply:
![image](https://github.com/geonetwork/core-geonetwork/assets/1701393/0adac292-e719-4f99-b697-b04efd84a7fe)